### ISSUE TITLE
[New instance] inv.odyssey346.dev

### ISSUE THREAD

**Issue by Odyssey346** [iv-org/documentation#270]
### URL

https://inv.odyssey346.dev

### Mandatory checks

- [X] Instance has a domain name
- [X] Instance is served over HTTPS
- [X] Statistics (`/api/v1/stats`) are enabled
- [x] Instance is properly configured (including the mandatory [post install configuration](https://docs.invidious.io/installation/#post-install-configuration))
- [X] Instance has an automatic hourly restart setup

### Maintainer chart

- [X] Ensure that my instance is up to date (less than one month old)
- [X] Ensure a proper uptime of my instance (around 90%)

### Host country

France

### Analytics

- [ ] They are GDPR/CCPA compliant
- [ ] This is stated in your Privacy Policy
- [ ] You have provided the source code's URL in the dedicated field above

### Additional information

I use caddy btw

**Comment by unixfox** [comment]
Instance added on the uptime tracker: https://stats.uptimerobot.com/89VnzSKAn/792576736

**Comment by unixfox** [comment]
Instance seems down, do you still want to host a public instance?

**Comment by Odyssey346** [comment]
Yes. Sorry, but the VPS that is running Invidious right now is down due to maintenance. I will wait for it to come back up so I can docker swarm them

**Comment by Odyssey346** [comment]
We are just waiting on OVH to respond because my VPS is hosted on a dedi, and it has to be fixed by someone at OVH

**Comment by Odyssey346** [comment]
I have heard from my provider that he might discontinue the Poland host. So I will run it on the new VPS for now, which means a reset.

**Comment by unixfox** [comment]
note: need to add https://github.com/iv-org/documentation/issues/271 with this instance too

**Comment by Odyssey346** [comment]
Well that's unfortunate... Seems like my host messed something up with my VPS! Just excuse this downtime, I have contacted him and I will let you know when I get a message. In the meantime, screw OVH

**Comment by Odyssey346** [comment]
And we're back

**Comment by unixfox** [comment]
Please consider joining our Matrix room for public instance maintainers by joining our Matrix room: [matrix.to/#/#invidious:matrix.org](https://matrix.to/#/#invidious:matrix.org)
then pinging @\unixfox, @\TheFrenchGhosty and @\SamantazFox for asking to be invited to the Matrix room.
We discuss troubles managing a public instance, sharing some advices and more.

**Comment by Odyssey346** [comment]
Im already in it

**Comment by unixfox** [comment]
Feel free to open a PR for that.

**Comment by unixfox** [comment]
@pistasjis 

Hello, your instance is affected by this new issue: https://github.com/iv-org/invidious/issues/3822

Please have a look for a temporary solution: https://github.com/iv-org/invidious/issues/3822#issuecomment-1565401621

**Comment by pistasjis** [comment]
Hi! I don't really have the time to set up IPv6 for Docker right now, and I can't change my IPv4.

**Comment by unixfox** [comment]
Hello,

We temporarily removed your instance in order to avoid the Invidious users to use an instance that doesn't work for the moment.

Once you have fixed the issue (`The video returned by YouTube isn't the requested one. (WEB client) (VideoNotAvailableException)`) on your instance, please ask to get your instance back in the list.

We are fully aware that this issue is not your fault, but for the moment we can't do anything than filtering in the list the instances that works from the instances that don't. Again, like said previously, we are working on the problem, and all progress will be posted in [#3822](https://github.com/iv-org/invidious/issues/3822).

Thank you for your understanding.

**Comment by pistasjis** [comment]
Do you know how to configure IPv6 for a network in Docker Compose? I've got IPv6 set up for Docker but the network isn't picking up on that, so I think I'll have to do some manual stuff. compose.yml is version 3

**Comment by unixfox** [comment]
Maybe this will help you, I've copied paste these comments from a matrix room:

> after enabling docker-ipv6nat because buyvm iirc only gives one v6
> 
> ```
> version: '3.3'
> services:
>     ipv6nat:
>         container_name: ipv6nat
>         privileged: true
>         network_mode: host
>         restart: unless-stopped
>         volumes:
>             - '/var/run/docker.sock:/var/run/docker.sock:ro'
>             - '/lib/modules:/lib/modules:ro'
>         image: robbertkl/ipv6nat
> ```

> and make ports: 7573:3000 to ports: 0.0.0.0:7573:3000

> had to replcae networks: invidious: with this

> ```
> networks:
>   invidious:
>     name: invidious
>     enable_ipv6: true
>     ipam:
>       config:
>         - subnet: fd00:dead:beec::/48
> ``
> ```

> i have enabled ipv6 on inv.bp.projectsegfau.lt

From @gi-yt

**Comment by unixfox** [comment]
I have updated my comment to provide an easy docker-compose file for adding ipv6: https://github.com/iv-org/invidious/issues/3822#issuecomment-1565401621

**Comment by pistasjis** [comment]
I'm going to look into this after I graduate, sorry for not responding.

**Comment by pistasjis** [comment]
Sorry again for not responding (jesus, 3 weeks already?), I'm not very up to date on this issue, does enabling IPv6 mitigate it still? Or have they started blocking IPv6 ranges as well?

**Comment by gi-yt** [comment]
It still does, though they started blocking individual v6 addresses for some bigger instances..

**Comment by pistasjis** [comment]
Great to hear. I'll look into this again

**Comment by pistasjis** [comment]
I've enabled IPv6 using the tutorial above, but it still doesn't work
![image](https://github.com/iv-org/documentation/assets/57069715/6f123853-3bbb-4ea4-a213-7981d08b8c4c)

**Comment by pistasjis** [comment]
![image](https://github.com/iv-org/documentation/assets/57069715/be0ea6f4-55c3-4038-a036-3b6a55e2b7fe)

Just to confirm the container has IPv6 access.

**Comment by pistasjis** [comment]
I just realized I had an entire /64 space allocated to my vps, lmao

**Comment by pistasjis** [comment]
My range might be blocked. I changed the IPv6 address from ``2001:41d0:203:2d1::1`` to ``2001:41d0:203:2d1::69``, and I still get this error. That sucks :/

**Comment by pistasjis** [comment]
![image](https://github.com/iv-org/documentation/assets/57069715/160c96c8-cbf5-4582-a7dd-680513dd3b6b)
 
and yes, YouTube is reachable via IPv6.

**Comment by unixfox** [comment]
Could you join our matrix room? Github issues are not really designed for "advanced" discussions.

**Comment by gi-yt** [comment]
set  force_resolve: ipv6

**Comment by pistasjis** [comment]
> set force_resolve: ipv6

In Invidious config? there is no such configuration option in my compose file

**Comment by pistasjis** [comment]
Okay, it was in Invidious config. The Invidious instance is back up and videos seem to load, so you can add my instance back to the list.

**Comment by jojobozo** [comment]
> Okay, it was in Invidious config. The Invidious instance is back up and videos seem to load, so you can add my instance back to the list.

I'm trying to add your instance on LibreTube and it requires the URL to the instance API, what is it?

**Comment by unixfox** [comment]
> > Okay, it was in Invidious config. The Invidious instance is back up and videos seem to load, so you can add my instance back to the list.
> 
> I'm trying to add your instance on LibreTube and it requires the URL to the instance API, what is it?

Please contact the maintainer separately through other means, this is not the correct place for discussing issues with adding an instance to libretube.
